### PR TITLE
feat(FE): TTL/s3 integration

### PR DESCRIPTION
**Pull request by pranshuchittora**

### CODE REVIEW

**Review by palash-signoz — CHANGES_REQUESTED** [review]

File: frontend/src/pages/AllAlertChannels/index.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ function AllAlertChannels(): JSX.Element {
 				routes: [
 					{
 						Component: GeneralSettings,
-						name: 'General Settings',
+						name: 'General',
```
[the remark this answers is not in the document]
Comment (pranshuchittora) [reply frontend/src/pages/AllAlertChannels/index.tsx:17]: Using translated string won't be a good idea over here bcoz we are using the path name for conditional check. 

Screenshot 2022-04-05 at 14 52 34

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx:17]: that's the string only _(with Axolo)_

[the remark this answers is not in the document]
Comment (pranshuchittora) [reply frontend/src/pages/AllAlertChannels/index.tsx]: Done

File: frontend/src/api/disks/getDisks.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import axios from 'api';
+import { ErrorResponseHandler } from 'api/ErrorResponseHandler';
+import { AxiosError } from 'axios';
+import { ErrorResponse, SuccessResponse } from 'types/api';
+import { IDiskType } from 'types/api/disks/getDisks';
+
+const getDisks = async (): Promise<
+	SuccessResponse<IDiskType[]> | ErrorResponse
```
[the remark this answers is not in the document]
Comment (pranshuchittora) [reply frontend/src/api/disks/getDisks.ts]: Done

File: frontend/src/pages/AllAlertChannels/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: <https://react.i18next.com/guides/multiple-translation-files> _(with Axolo)_

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: @pranshuchittora I think we need to go for this approach _(with Axolo)_

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: or we can pick in later release ? _(with Axolo)_

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: what do you suggest ? _(with Axolo)_

[the remark this answers is not in the document]
Comment (pranshuchittora) [reply frontend/src/pages/AllAlertChannels/index.tsx]: Yep need to migrate to this and also we need to rename the `translation.json`  to `en.json` . Let’s pick this in the next release as there will be lot of rework :thinking_face: Thoughts ?? _(with Axolo)_

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: ok :white_check_mark: _(with Axolo)_

[the remark this answers is not in the document]
Comment (palash-signoz) [reply frontend/src/pages/AllAlertChannels/index.tsx]: yes this is a separate issue can be picked later and i am merging this PR _(with Axolo)_

**Review by palash-signoz — APPROVED** [review]

**Pull request merged**